index.d.ts

```diff
@@ -797,7 +797,9 @@ declare namespace Eris {
     user: User;
   }
   interface PartialUser {
+    accentColor: number | null;
     avatar: string | null;
+    banner: string | null;
```

Comment:
At the moment both of these are partial nullables as they don't seem to be coming through on gateway, only via REST